lib/author.rb

```diff
@@ -0,0 +1,26 @@
+class Author
+  attr_reader :first_name,
+              :last_name,
+              :books
+
+  def initialize(
+    first_name:,
+    last_name:
```

Comment:
This should be on one line.

Reply:
This should also only accept one argument, not two. Recheck your syntax on how initialize works